proto/requests.proto

```diff
@@ -188,3 +189,29 @@ message GetAccountProofsRequest {
     // all requested accounts.
     repeated digest.Digest code_commitments = 3;
 }
+
+// Returns a list of unconsumed network notes using pagination.
+message GetUnconsumedNetworkNotesRequest {
+    // Page number to retrieve.
+    uint64 page = 1;
+
+    // Number of notes to retrieve per page.
+    uint64 limit = 2;
+}
+
+// Creates a new network note.
+message CreateNetworkNoteRequest {
+    // Plain note.
+    note.Note note = 1;
+    
+    // Id of the transaction that created the note.
+    digest.Digest transaction_id = 2;
+}
+
+message UpdateNetworkNoteRequest {
+    // Transaction id of the note to update.
+    digest.Digest transaction_id = 1;
+
+    // New status of the note.
+    note.NetworkNoteStatus status = 2;
+}
```

Comment:
I may have missed something - but it is not clear to me how we plan to use these requests.